Svc/FpySequencer/FpySequencerDirectives.cpp

```diff
@@ -1075,41 +1110,60 @@ Signal FpySequencer::allocate_directiveHandler(const FpySequencer_AllocateDirect
     return Signal::stmtResponse_success;
 }
 
-Signal FpySequencer::storeConstOffset_directiveHandler(const FpySequencer_StoreConstOffsetDirective& directive,
+Signal FpySequencer::storeLocalConstOffset_directiveHandler(const FpySequencer_StoreLocalConstOffsetDirective& directive,
                                                        DirectiveError& error) {
     if (this->m_runtime.stack.size < directive.get_size()) {
         // not enough bytes to pop
         error = DirectiveError::STACK_ACCESS_OUT_OF_BOUNDS;
         return Signal::stmtResponse_failure;
     }
-    Fpy::StackSizeType stackOffset = this->m_runtime.stack.lvarOffset() + directive.get_lvarOffset();
-    // if we popped these bytes off, and put them in lvar array, would we go out of bounds
-    if (stackOffset + directive.get_size() > this->m_runtime.stack.size - directive.get_size()) {
-        // write into lvar array would go out of bounds
+    // Calculate the actual address using signed offset from stack_frame_start
+    I32 lvarOffset = directive.get_lvarOffset();
```

Comment:
We really need to be consistent on these types.   I see `Fpy::StackSizeType`, `I32`, and now `I64` for address.